src/checkout-buttons/strategies/masterpass-button-strategy.ts

```diff
@@ -0,0 +1,144 @@
+import { CheckoutButtonInitializeOptions, CheckoutButtonOptions } from '../';
+import { Checkout, CheckoutActionCreator, CheckoutStore } from '../../checkout';
+import {
```

Comment:
🍹 In this case, you can put them in one single line.

Reply:
![image](https://user-images.githubusercontent.com/6343437/47298273-f5e8db80-d5dc-11e8-9ff3-c4c4c808c2a9.png)
i thin is too long here in one line

Reply:
i think the line is too long for that:
![image](https://user-images.githubusercontent.com/6343437/47298584-ca1a2580-d5dd-11e8-96e7-c656f8b1d4e0.png)